controllers/k8sgpt_controller.go

```diff
@@ -213,7 +213,7 @@ func (r *K8sGPTReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctr
 			return r.finishReconcile(err, false)
 		}
 		// Update metrics count
-		if k8sgptConfig.Spec.AI.Enabled {
+		if k8sgptConfig.Spec.AI.Enabled && len(response.Results) > 0 {
```

Comment:
what if len(response.Results) is 2, is k8sgpt call just 1 request? or 2 request?

Reply:
> what if len(response.Results) is 2, is k8sgpt call just 1 request? or 2 request?

The results could be 10,200,1000.. the line 
```
response, err := k8sgptClient.ProcessAnalysis(deployment, k8sgptConfig)
```

Is a single GRPC call that returns a results array